gui/slick/css/style.css

```diff
@@ -466,59 +467,51 @@ home.mako
     overflow: visible;
     text-align: center;
     text-shadow: 0 0 0.1em rgb(255, 255, 255);
-    vertical-align: middle;
```

Comment:
They get disabled because of other rules so removing it doesn't change anything. If it does break anything then we need to actually fix it instead of just adding it back since it's incorrect and out of spec.